src/alfasim_sdk/_internal/alfacase/schema.py

```diff
@@ -97,12 +139,27 @@
 )
 environment_property_description_schema = Map(
     {
-        "position": Map({"value": Float(), "unit": Str()}),
-        "temperature": Map({"value": Float(), "unit": Str()}),
+        "position": UnsafeOrValidator(
+            Map({"value": Float(), "unit": Str()}),
+            Map({"value": Str(), "unit": Str()}),
+        ),
+        "temperature": UnsafeOrValidator(
+            Map({"value": Float(), "unit": Str()}),
+            Map({"value": Str(), "unit": Str()}),
+        ),
```

Comment:
```suggestion
        "position": UnsafeOrValidator(
            Map({"value": Float(), "unit": Str()}),
            Map({"expr": Str(), "unit": Str()}),
        ),
        "temperature": UnsafeOrValidator(
            Map({"value": Float(), "unit": Str()}),
            Map({"expr": Str(), "unit": Str()}),
        ),
```
If we are using just "value" we don't need `UnsafeOrValidator`. Just:

```suggestion
        "position": Map({"value": Float() | Str(), "unit": Str()}),
        "temperature": Map({"value": Float() | Str(), "unit": Str()}),
```